app/views/schools/contacts/_form.html.erb

```diff
@@ -4,7 +4,7 @@
 
 
 <% if f.object.user.nil? %>
-  <%= f.input :staff_role_id, label: t('schools.contacts.form.labels.role'), collection: StaffRole.order(:title) %>
+  <%= f.input :staff_role_id, label: t('schools.contacts.form.labels.role'), collection: StaffRole.all.map { |staff_role| [staff_role.translated_title, staff_role.id] }.sort %>
```

Comment:
We've got a few other places with the same staff role dropdown - wonder if the collection translation could be factored out into a helper..?